libs/datasets/projections_schema.py

```diff
@@ -0,0 +1,31 @@
+OUTPUT_COLUMN_REMAP = {
```

Comment:
What's the input/output here? like what datasets are these mapping between?

Reply:
updated the names